Install Go 번역

MagicalLas opened the pull request:
기존에는 install-go.md 파일에 번역된 내용이 존재하지 않았습니다.
따라서 원 글을 번역하여 install-go.md 파일에 추가하였습니다.

원래 @MagicalLas는 #1이 아니라 #2을 번역해야 하지만 이슈에 잘못 링크가 되어 #1을 번역하였습니다. [코멘트](https://github.com/MiryangJung/learn-go-with-tests-ko/issues/2#issuecomment-801215154) 에서 논의한 대로 #1의 PR을 보내도록 하였습니다.

See Also
  - #1, #2
  - https://github.com/MiryangJung/learn-go-with-tests-ko/blob/master/install-go.md

MagicalLas committed:
Install Go 번역!

기존에는 install-go.md 파일에 번역된 내용이 존재하지 않았습니다.
따라서 원 글을 번역하여 install-go.md 파일에 추가하였습니다.

원래 @MagicalLas는 #1이 아니라 #2을 번역해야 하지만 이슈에 잘못 링크가 되어 #1을 번역하였습니다. [코멘트](https://github.com/MiryangJung/learn-go-with-tests-ko/issues/2#issuecomment-801215154) 에서 논의한 대로 #1의 PR을 보내도록 하였습니다.

See Also
  - #1, #2
  - https://github.com/MiryangJung/learn-go-with-tests-ko/blob/master/install-go.md

MiryangJung commented:
현재 [번역 규칙](https://github.com/MiryangJung/learn-go-with-tests-ko#%EB%B2%88%EC%97%AD-%EA%B7%9C%EC%B9%99) 이 바뀌었습니다.  
- `다` 를 맺음말로 사용합니다. [예시](https://ultimate-go-korean.github.io/translation/)

변경하여 다시 pull 해주시면 감사하겠습니다.

MagicalLas committed:
문장 어미 `~요`를 `~다`로 변경

See Also
  - https://github.com/MiryangJung/learn-go-with-tests-ko/pull/50#issuecomment-801530863

MagicalLas committed:
번역이 안된 문장 추가

MagicalLas commented:
@MiryangJung 변경하였습니다.

> 변경하여 다시 pull 해주시면 감사하겠습니다.

혹시 다시 pull을 하라는 의미는 pull request를 닫고 새로운 pull request를 만들라는 의미시인가요?

MiryangJung commented:
앗 아뇨 그냥 커밋만해주시면됩니다. 제가 헷갈리게 적었네요.

MiryangJung commented:
맺음말을 `다` 로 변경해주시면 감사하겠습니다. [예시](https://miryang.gitbook.io/learn-go-with-tests/go/iteration)

MagicalLas committed:
apply translation convention

MagicalLas commented:
변경했어요!
